spi/identity-hub-spi/src/main/java/org/eclipse/edc/identityhub/spi/credentials/model/CredentialEnvelope.java

```diff
@@ -33,6 +36,10 @@ public interface CredentialEnvelope {
      * @param mapper The json mapper.
      * @return The result of the conversion process
      */
+    Result<List<VerifiableCredential>> toVerifiableCredentials(ObjectMapper mapper);
 
-    Result<VerifiableCredential> toVerifiableCredential(ObjectMapper mapper);
+    /**
+     * Returns the root JWT that implementor of {@link CredentialEnvelope}.
+     */
+    SignedJWT getJwt();
```

Comment:
There should be not JWT mention in this interface. As this will also be used as root object for the VC/VP not in JWT format.

Reply:
It's done.